packages/core/src/internal/services/i18n.service.ts

```diff
@@ -58,6 +58,12 @@ export const componentStringsDefault = {
     contentBox: 'Scrollable Modal Body',
     contentEnd: 'End of Modal Content',
   },
+  navigation: {
+    navigationElement: 'navigation',
+    navigationLabel: 'navigation menu',
+    navigationAbridgedText: 'View abridged menu',
```

Comment:
A11y decided on the word abridged rather than expanded. I didn't change the attribute tho, it is still expanded so abridged  expanded